tasmota/xdrv_05_irremote.ino

```diff
@@ -176,7 +176,10 @@ void IrSendInit(void)
 \*********************************************************************************************/
 
 const bool IR_RCV_SAVE_BUFFER = false;         // false = do not use buffer, true = use buffer for decoding
-const uint32_t IR_TIME_AVOID_DUPLICATE = 500;  // Milliseconds
+
+#ifndef IR_TIME_AVOID_DUPLICATE
+#define IR_TIME_AVOID_DUPLICATE = 50           // Milliseconds
```

Comment:
It is not needed the equal sign in a #define

Just delete the sign to pass the CI tests